projects/alfresco/rds/output.tf

```diff
@@ -103,3 +98,13 @@ output "rds_db_instance_port" {
   description = "The database port"
   value       = "${module.db_instance.db_instance_port}"
 }
+
+output "rds_db_instance_endpoint_cname" {
+  description = "The connection endpoint"
+  value       = "${aws_route53_record.rds_dns_entry_public.fqdn}"
```

Comment:
Do we really want our database on a public endpoint, surely this will force traffic to flow out the IG and back in